### PR TITLE
Review your project from trials!

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we:
- Added index.html file
- Added style.css file
- Added index.js file
- Added babel files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <h1>JS Documentation by Onyebuchi_Fombi</h1>
+        </header>
+        <ol id="nav-list">
+            <a class="nav-link" href="#Background"> <li>Background</li> </a>
+            <a class="nav-link" href="#What_you_should_already_know"> <li>What you should already know</li> </a>
+            <a class="nav-link" href="#JavaScript_and_Java"> <li>JavaScript and Java</li> </a>
+            <a class="nav-link" href="#Hello_world"> <li>Hello world</li> </a>
+            <a class="nav-link" href="#Variables"> <li>Variables</li> </a>
+        </ol>
+    </nav>
```
Comment (Fombi-Favour) [inline index.html:23]: Perfectly used the semantic HTML tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (Fombi-Favour) [inline index.html:79]: Please kindly remove the words that does not define a portion in your code.

**Review by Fombi-Favour — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make all required changes.

Submit for a re-review when you're done.

Thanks 😁

File: index.html (hunk 2 above)
Comment (Ibizugbe) [reply index.html:79]: Nice job using `github` inline comments feature on your comments👍. But let's make this comment even better by stating why you are requesting the change😉. remember you may be aware of the reason but not for someone else😉. don't foget we learnt this in our lesson on [how to give a code review😉.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md)

image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Fombi-Favour 👋👋, 

tryagain


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!👍

## Highlights

- [x] Nice job on your comments👍.
- [x] Nice job making use of github inline comment feature👍.
- [x] The student has submitted a review in the Pull Request they opened utilizing the "Review changes" feature from GitHub👍.

## Required Changes ♻️

- [ ] Kindly note that you're required to add atleast two comments that points to a piece of code that should have been done better👍 You have only added one comment to make improvements and the other is a compliment😉 Kindly add atleast one comment that shows how the project should have been done better👍
done



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] I suggest you add some gifs or emojis to make your comments more humanly friendly😉.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Fombi-Favour) [reply index.html:79]: Please kindly remove the words that does not define a portion in your code as comments. Those words as comments makes your code look unprofessional.

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (Fombi-Favour) [inline js/index.js:27]: Please kindly remove the words that does not define a portion in your code as comments 😁. Those words as comments makes your code look unprofessional.

**Review by Fombi-Favour — COMMENTED** [review]
# Changes required ♻️ 

You're that close to be approved 👍 

Please check my comments below and make all required changes.

Submit for a re-review when you're done.

Thanks 😁

**Review by divinecharlotte — APPROVED** [review]
# Hello @Fombi-Favour    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- good communication while giving feedback👍 





## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**